edge-modules/azure-monitor/src/README.md

```diff
@@ -53,6 +53,10 @@ Optional config items:
     - This should really only be turned off in extremely CPU-bound applications, if at all.
     - ex: `true`
     - Defaults to true.
+- `TransformForUpload`
+    - If the metrics data needs to be flattened prior to published as IoT messages.
```

Comment:
I can imagine that people will ask, "Why would I want to flatten the data prior to publishing as IoT messages?" Do we need to explain that here (or somewhere)? We're flattening here for a very specific purpose, if I understand correctly...

Reply:
will do

Reply:
Agree with Damon. Perhaps, change the env var name to "TransformForIoTCentral" with a description of "Transform a subset of metrics in an IoT Central friendly format".

I would also check `UploadTarget` is set to `IotMessage` if this env var is true.